src/utils/verifyPlainObject.js

```diff
@@ -1,4 +1,4 @@
-import isPlainObject from 'lodash/isPlainObject'
+import isPlainObject from './isPlainObject'
```

Comment:
This looks good to me and I'm definitely a fan of removing the lodash dependency, but I wonder if instead of rolling our own isPlainObject we should bring in https://github.com/jonschlinkert/is-plain-object ?

Reply:
Also, I rejected `is-plain-object` because it uses the `toString()` method, which is relatively expensive.